### ISSUE TITLE
Make coverage unit test files location and pattern configurable

### ISSUE THREAD

**Issue by sagiegurari** [sagiegurari/cargo-make#51]
Based on @azriel91 feedback in #44 to prevent issues when running interactive executables.

**Comment by azriel91** [comment]
I'm thinking that it could be better to do this in Rust than in bash, assuming that it is true that the `target/debug/<crate_name>-ha5hc0de` executables are strictly a set of all tests in the workspace, then something along the lines of this might work:

```rust
// pseudo-rust, probably doesn't compile since I wrote it in a scratchy pad
use std::env;
use std::ffi::OsStr;
use std::path::Path;

fn coverage_executables(workspace_crates: Iterator<&str>) -> Vec<Path> {
    let profile = env::var("PROFILE").unwrap_or("debug"); // probably should be an expect("..")
    let target_dir = Path::new("target").join(profile);
    path.read_dir().expect("failed to read target/<profile> directory") // TODO: format string
        .filter_map(|entry| entry.ok())
        .filter(|entry| {
            if let Ok(file_type) = entry.file_type() {
                return file_type.is_file();
            }
            false
        })
        .filter(|entry|
            let file_name = entry.path()
                    .iter()
                    .last()
                    .to_string_lossy();
            workspace_crates.any(|krate| format!("{}-[a-f0-9]+", krate).matches(file_name)))
        .collect();
}
```

**Comment by sagiegurari** [comment]
Fixed in 0.9.2, all tests will now run and no configuration is needed or special naming convesions.
